planning/behavior_velocity_planner/include/utilization/util.hpp

```diff
@@ -124,6 +138,90 @@ geometry_msgs::msg::Pose transformRelCoordinate2D(
   const geometry_msgs::msg::Pose & target, const geometry_msgs::msg::Pose & origin);
 geometry_msgs::msg::Pose transformAbsCoordinate2D(
   const geometry_msgs::msg::Pose & relative, const geometry_msgs::msg::Pose & origin);
+SearchRangeIndex getPathIndexRangeWithLaneId(
+  const autoware_auto_planning_msgs::msg::PathWithLaneId & path,
+  const geometry_msgs::msg::Point & point, const int64_t lane_id,
+  const double distance_threshold = 10.0);
```

Comment:
https://github.com/autowarefoundation/autoware.universe/pull/405#discussion_r815623270
Which one corresponds to the function described here?

Reply:
```
SearchRangeIndex getSearchRangeIndexWithLaneId(
  const autoware_auto_planning_msgs::msg::PathWithLaneId & path,
  const geometry_msgs::msg::Point & point, const int64_t lane_id)
```

Reply:
same as https://github.com/autowarefoundation/autoware.universe/pull/405#discussion_r819267421